elyra/metadata/schemasproviders.py

```diff
@@ -77,6 +78,7 @@ class RuntimesSchemas(ElyraSchemasProvider):
     def get_schemas(self) -> List[Dict]:
 
         kfp_needed = False
+        airflow_schema_present = False
```

Comment:
Could we please rename `kfp_needed` to `kfp_schema_present` since these flags have similar meanings?